### PR TITLE
Cleanup 'update_kernel' and add another workaround for Update_3

### PR DESCRIPTION
**Pull request by mimi1vx**
- Verification run: http://quasar.suse.cz/tests/864

### CODE REVIEW

File: tests/kernel/update_kernel.pm [hunk 1]
```diff
@@ -19,6 +19,8 @@ use utils;
 use version_utils 'is_sle';
 use qam;
 use kernel 'remove_kernel_packages';
+use power_action_utils 'power_action';
+
```
Comment (czerw) [inline tests/kernel/update_kernel.pm]: + empty line?

Comment (mimi1vx) [reply tests/kernel/update_kernel.pm]: and ?

Comment (czerw) [reply tests/kernel/update_kernel.pm]: two(instead of one) empty lines between lines 22 and 25

Comment (dzedro) [reply tests/kernel/update_kernel.pm]: @mimi1vx is sulky :wink:

**Pull request merged**